nmrfx-analyst/src/main/java/org/nmrfx/analyst/peaks/Multiplets.java

```diff
@@ -1132,20 +1130,14 @@ public static void splitRegionsByPeakCount(Set<DatasetRegion> regions, PeakList
         }
         Dataset dataset = Dataset.getDataset(peakList.getDatasetName());
         double[][] limits = new double[1][2];
-        Set<DatasetRegion> newRegions = new TreeSet<>();
+        List<DatasetRegion> newRegions = new ArrayList<>();
         while (true) {
-            regions.stream().forEach(region -> {
+            regions.forEach(region -> {
                 limits[0][0] = region.getRegionStart(0);
                 limits[0][1] = region.getRegionEnd(0);
-                DatasetRegion newRegion = null;
                 List<Peak> peaks = locatePeaks(peakList, limits, dim);
                 if (peaks.size() > maxPeaks) {
-                    List<PeakDim> peakDims = new ArrayList<>();
-                    peaks.forEach((peak) -> {
-                        peakDims.add(peak.peakDims[0]);
-                    });
                     int nSplits = peaks.size() / maxPeaks;
-                    peakDims.sort(comparing(PeakDim::getChemShiftValue));
```

Comment:
Sonar lint wanted lambda to be shorter, doesn't seem like peakDims is used for anything so removed it. This entire method splitRegionsByPeakCount is unused though...